### ISSUE TITLE
NameError on python3.5

### ISSUE THREAD

**Issue by ndclt** [achillesrasquinha/pipupgrade#32]
When launching pipupgrade (on version 1.5.1) on python 3.5, during the import, I have an error with the following stacktrace:
```
Traceback (most recent call last):
  File ".tox/py35-unit/bin/pipupgrade", line 6, in <module>
    from pipupgrade.__main__ import main
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/__init__.py", line 7, in <module>
    from pipupgrade.__main__ import main
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/__main__.py", line 5, in <module>
    from pipupgrade.commands import command as main
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/commands/__init__.py", line 7, in <module>
    from pipupgrade.model         import Project, Package, Registry
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/model/__init__.py", line 3, in <module>
    from pipupgrade.model.package  import Package
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/model/package.py", line 5, in <module>
    from pipupgrade import _pip, request as req, db
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/_pip.py", line 2, in <module>
    from pipupgrade._compat import iteritems
  File ".tox/py35-unit/lib/python3.5/site-packages/pipupgrade/_compat.py", line 45, in <module>
    ModuleNotFoundError = ModuleNotFoundError
NameError: name 'ModuleNotFoundError' is not defined
```

I look in the code associated to the version 1.5.1, I cannot found this line ([_compat.py](https://github.com/achillesrasquinha/pipupgrade/blob/e69dbb954b33dc89146326d0241779cff1bdbc3c/src/pipupgrade/_compat.py))

I workaround this by using the version 1.5.0.

**Comment by ndclt** [comment]
When installing `pipupgrade` from the tagged commit (e69dbb954b33dc89146326d0241779cff1bdbc3c), I don't have the matter. Maybe the code pushed on pypi is not the correct commit?

**Comment by achillesrasquinha** [comment]
This seems to be fixed on master. https://github.com/achillesrasquinha/pipupgrade/blob/master/src/pipupgrade/_compat.py#L44

**Comment by ndclt** [comment]
Perfect, thanks. Can you publish a new patch version?

**Comment by CMiksche** [comment]
This error still exists. Installed the package today:
```
# pipupgrade --check
Traceback (most recent call last):
  File "/usr/local/bin/pipupgrade", line 7, in <module>
    from pipupgrade.__main__ import main
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/__init__.py", line 7, in <module>
    from pipupgrade.__main__ import main
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/__main__.py", line 5, in <module>
    from pipupgrade.commands import command as main
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/commands/__init__.py", line 7, in <module>
    from pipupgrade.model         import Project, Package, Registry
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/model/__init__.py", line 3, in <module>
    from pipupgrade.model.package  import Package
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/model/package.py", line 5, in <module>
    from pipupgrade import _pip, request as req, db
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/_pip.py", line 2, in <module>
    from pipupgrade._compat import iteritems
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/_compat.py", line 45, in <module>
    ModuleNotFoundError = ModuleNotFoundError
```

**Comment by achillesrasquinha** [comment]
@CMiksche could you try version 1.5.2? Use `pipupgrade --self` to update pipupgrade.

**Comment by CMiksche** [comment]
> @CMiksche could you try version 1.5.2? Use `pipupgrade --self` to update pipupgrade.

I have tried it, same error:

```
Successfully installed pipupgrade-1.5.2
root@orangepipc:/home/orangepi# pipupgrade --check
Traceback (most recent call last):
  File "/usr/local/bin/pipupgrade", line 7, in <module>
    from pipupgrade.__main__ import main
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/__init__.py", line 7, in <module>
    from pipupgrade.__main__ import main
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/__main__.py", line 5, in <module>
    from pipupgrade.commands import command as main
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/commands/__init__.py", line 9, in <module>
    from pipupgrade.commands.helper import (
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/commands/helper.py", line 7, in <module>
    from pipupgrade.model               import Registry
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/model/__init__.py", line 3, in <module>
    from pipupgrade.model.package  import Package
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/model/package.py", line 6, in <module>
    from pipupgrade import _pip, request as req, db, log
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/_pip.py", line 2, in <module>
    from pipupgrade._compat import iteritems
  File "/usr/local/lib/python3.5/dist-packages/pipupgrade/_compat.py", line 48, in <module>
    ModuleNotFoundError = ModuleNotFoundError
NameError: name 'ModuleNotFoundError' is not defined
```

**Comment by achillesrasquinha** [comment]
Fixed with `1.6.2`!

Use the following command to upgrade `pipupgrade`
`pipupgrade --self`
